src/orchestrator/orchestrator/orchestrator.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import codecs
+import datetime
+import logging
+from time import sleep
+import jsonpickle
+
+from learner.learner import learn_for_users
+from scraper.scraper import scrap
+from server.frontendstructs import Document
+from topicmodeller.topicmodeller import TopicModeller, classify
+
+
+logging.basicConfig(format=u'%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO, ilename='orchestrator.log')
+
```

Comment:
I think this line won't be executed as it's a package and not a script, I would create a setup_env method and call it from orchestrate()